packages/rich-text/src/plugins/Tracking/utils.ts

```diff
@@ -6,7 +6,8 @@ import { PlateEditor } from '../../internal/types';
 
 export function getCharacterCount(editor: PlateEditor) {
   const document = contentfulSlateJSAdapter.toContentfulDocument({
-    document: editor.children,
+    // eslint-disable-next-line -- parameter type is not exported @typescript-eslint/no-explicit-any
```

Comment:
`toContentfulDocument` expects a list of `SlateNode` but we pass a list of `Element`. The two are not compatible. It didn’t surface before because the types in `contentful-slatejs-adapter` weren’t working.